### PR TITLE
Assignment4 @Matt Pennock

### PR DESCRIPTION
**Pull request by DakotaCondos**

### CODE REVIEW

**Comment by DakotaCondos** [comment]
Link to website: https://polite-dune-09ecd591e.1.azurestaticapps.net/

**Comment by PlantBrush** [comment]
I will review this.

**Comment by kelseymcmahon** [comment]
The web app is working and looks good. The daily word seems to work and I was able to play and add my score. It would be nice to see more than one day of scores on the Daily Word scoreboard, maybe add some data there. The mobile view on the abut page looks a little strange with the two columns, maybe push them into one column, with the image on top of the text.

**Comment by PlantBrush** [comment]
It looks good and seems to be mostly working fine. ~~I did notice some strange behavior with how it stores players. I was able to submit the same results multiple times by clicking on the submit button again, which artificially inflates my score in the leaderboard.~~ Sometimes it would add a new record to the player list for the same name, when people with the same name should instead be listed under the same entry, from my understanding. Additionally, the game list is not updating the number of total plays as it should. I also noticed that some words that are picked as options for the game are not showing up as valid in the "available words". Also, for me, the desktop keyboard is worse than the mobile one. The buttons are overlapping on 1920x1080 for me.

**Review by mmwoodfo — COMMENTED** [review]
You have some pretty linter errors, make sure you run your linter before the final submission date:
image

**Comment by mmwoodfo** [comment]
Good job with some of the extra credit.

File: Wordle.Api/Services/GameService.cs [hunk 1]
```diff
@@ -1,147 +1,133 @@
...
-            if (_cache.TryGetValue(date, out var word))
-            {
-                return word;
-            }
-            DateWord? wordOfTheDay = _context.DateWords
-                .Include(x => x.Word)
-                .FirstOrDefault(dw => dw.Date == date);
+                var player = context.Players.First(x => x.Name == "Knights who say Ni");
+                var dateWord = context.DateWords.First(x => x.WordId == 92);
+                var score = context.ScoreStats.First(x => x.Seconds == 42);
```
[the remark this answers is not in the document]
Comment (mpennock5) [reply Wordle.Api/Services/GameService.cs:60]: There was supposed to be a ScoreStatsServervice.Seed(context); at line 41 in program.cs. It must of gotten removed by mistake when resolving the merge conflicts.

[the remark this answers is not in the document]
Comment (mmwoodfo) [reply Wordle.Api/Services/GameService.cs:60]: Got it - it didn't impact your score significantly, I think you'll still be happy when I post scores.

**Pull request closed without merging**